.github/workflows/release-integration.yml

```diff
@@ -58,34 +58,36 @@ jobs:
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above.
     steps:
       - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: actions/download-artifact@v2
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
 
       - name: Generate Docker image version from git tag
         run: |
           echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
           DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
           echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
-      - if: ${{ github.event.release.prerelease }}
+
+      - name: Change Docker image version if this run is a pre-release
+        if: ${{ github.event.release.prerelease }}
         run: |
           echo "DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG}-pre" >> $GITHUB_ENV
-      - uses: docker/setup-qemu-action@v1
-      - uses: docker/setup-buildx-action@v1
-
-      - name: Download all artifacts from build job
-        uses: actions/download-artifact@v2
-
-      - uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
-          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
 
       - name: Build and push Docker image
+        if: ${{ github.event.release.prerelease }}
         run: |
           docker buildx build --push --platform=$DOCKER_PLATFORMS \
             -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG \
             .
-      - name: Push :latest image
+
+      - name:  Build and push Docker image with :latest
```

Comment:
description is not correct

Reply:
Do you mean the double space or something wrong with the title?

Reply:
I mean that here we push both latest and the current tag since it is not the pre-release and above we push only the current tag.

Reply:
[Fixed](https://github.com/newrelic/newrelic-k8s-metrics-adapter/blob/0264a4a81c1f8b0b8d7092c78cf6fac92d08ca3b/.github/workflows/release-integration.yml#L75-L89).